nav2_controller/dwb_critics/src/oscillation.cpp

```diff
@@ -168,9 +172,9 @@ bool OscillationCritic::resetAvailable()
       return true;
     }
   }
-  if (oscillation_reset_time_ >= 0.0) {
+  if (oscillation_reset_time_ >= nav2_util::durationFromSeconds(0.0)) {
```

Comment:
You added the nav2_util::durationFromSeconds function, so perhaps should make sure everyone uses this across the packages?

Reply:
I searched through the repo looking for all places where seconds are converted to durations and changed them. Do you know of any that I missed?

Reply:
In my next PR I have some more code using Duration, so I just wanted to check that we're all using the same thing. Also related, what branch is the rclcpp repo on in our stack? The [rclcpp::Time](https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/time.hpp) has a function seconds() in addition to nanoseconds() in the master branch which was not in the bouncy release. In the meantime, we could add another util function for that conversion.

Reply:
Currently we are using bouncy latest. Sounds like it is time to switch to master. Yes on adding a function. We can then replace it with the rclcpp interface when we've switched.